src/js/components/DataChart/__tests__/DataChart-test.tsx

```diff
@@ -5,15 +5,16 @@ import 'jest-styled-components';
 import { Grommet } from '../../Grommet';
 import { Box } from '../../Box';
 import { Text } from '../../Text';
-import { DataChart } from '..';
+import { DataChart, DataChartProps } from '..';
+import { ChartProps } from '../../Chart';
 
 const data = [
   { a: 1, b: 'one', c: 111111, d: '2020-06-24' },
   { a: 2, b: 'two', c: 222222, d: '2020-06-23' },
 ];
 
 describe('DataChart', () => {
-  let warnSpy;
+  let warnSpy: jest.SpyInstance;
```

Comment:
I'm seeing the following error on this line: `Namespace 'jest' has no exported member 'SpyInstance'.`

If you're aren't seeing this error can you make sure your branch is updated with master and then run `yarn install` again? We recently did some jest dependency upgrades

Reply:
Just ran `yarn install` again, are you sure you have `@types/jest` installed?

Reply:
Just checked and I have v29.1.0 of `@types/jest` installed. I can check tomorrow if someone else on my team is seeing the error or not and then look into it more from there

Reply:
Interesting, just verified I have the same version of `@types/jest`, FWIW I'm on node `v18.10.0`

Reply:
I tested this with some other people today but we were getting different behavior across our environments. One of us was seeing the error, one was not, and the other was getting a different error. I am on node v14.9.2 so I am going to try and upgrade to amore recent version of node and see if that changes anything.

Reply:
I had my visual studio code using a different version of typescript than the project. I changed to use the project version and all looks good now. Sorry for the delay on this!